SANS_Critical_AI_Security_Guidelines_v1.1.md

```diff
@@ -55,6 +56,8 @@ Effective access controls are fundamental to securing AI modes, their associated
 
 > We've seen AI access controls range from hard-coded privileges to zero trust principles, especially as access extends beyond individual users to connected devices, applications, APIs, and other systems. As you deploy AI capabilities throughout your organization, we recommend utilizing least privilege principles and integrating with your current authentication mechanisms. However, if you plan to move toward stronger access controls, such as zero trust, consider AI in this transition.
 
+One of the most effective strategies for mitigating security risks is adhering to the principle of least privilege. This principle ensures that only a minimal set of users, devices, or systems are granted access to the training data, AI models, and associated resources, and only to the extent necessary to perform their specific roles. By limiting access to essential personnel or systems, organizations can significantly reduce the attack surface and the risk of accidental or malicious exposure of sensitive data.
+
```

Comment:
Leaving this part out of the main paper, as it is just explaining what Zero Trust is. Moving Zero Trust to a glossary term.